apps/betterangels/src/app/(private-screens)/add-note/SubmittedModal.tsx

```diff
@@ -0,0 +1,93 @@
+import { XmarkIcon } from '@monorepo/expo/shared/icons';
+import { Colors, Spacings } from '@monorepo/expo/shared/static';
+import { TextBold, TextRegular } from '@monorepo/expo/shared/ui-components';
+import { Image, Pressable, StyleSheet, View } from 'react-native';
+import Modal from 'react-native-modal';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import AnimatedIcon from './AnimatedIcon';
+
+interface ISubmittedModalProps {
+  closeModal: () => void;
+  isModalVisible: boolean;
+  firstName: string | null | undefined;
+}
+
+export default function SubmittedModal(props: ISubmittedModalProps) {
+  const { closeModal, isModalVisible, firstName } = props;
+
+  const insets = useSafeAreaInsets();
+  const bottomOffset = insets.bottom;
+  const topOffset = insets.top;
+
+  return (
+    <Modal
+      style={{
+        margin: 0,
+        flex: 1,
+        justifyContent: 'flex-end',
+        backgroundColor: Colors.WHITE,
+      }}
+      isVisible={isModalVisible}
+      onBackdropPress={closeModal}
+    >
+      <AnimatedIcon />
+
+      <View
+        style={{
+          flex: 1,
+          borderTopLeftRadius: 8,
+          borderTopRightRadius: 8,
+          paddingTop: topOffset + Spacings.xs,
+          paddingHorizontal: Spacings.md,
+          paddingBottom: 35 + bottomOffset,
+          backgroundColor: Colors.WHITE,
+          alignItems: 'center',
+          justifyContent: 'center',
+          position: 'relative',
+        }}
+      >
+        <Pressable
+          style={{
+            position: 'absolute',
+            top: Spacings.md,
+            right: Spacings.sm,
+            padding: Spacings.sm,
+          }}
+          accessible
+          accessibilityHint="closes the modal"
+          accessibilityRole="button"
+          accessibilityLabel="close"
+          onPress={closeModal}
+        >
+          <XmarkIcon size="md" color={Colors.BLACK} />
+        </Pressable>
+
+        <View style={styles.modalOverlay}>
+          <TextBold color={Colors.PRIMARY_DARK} size="xl">
+            You are such an Angel!
+          </TextBold>
+          <Image
+            accessibilityIgnoresInvertColors
+            source={require('../../assets/images/transparent-logo.png')}
+            style={{ width: 120, height: 115, marginTop: 67, marginBottom: 50 }}
```

Comment:
Is there a fe convention for saving custom values like this to constants? Or are magic numbers ok https://en.wikipedia.org/wiki/Magic_number_(programming)

Reply:
too specific numbers thats why I put straight numbers. don't want to create a constant that we are going to use only once. for me its shorter to write 67 than SIXTY_SEVEN

Reply:
ya, it wouldn't be `SIXTY_SEVEN`, I was just wondering if there's convention for stuff like this. I guess it wouldn't make sense since these aren't actually magic numbers. That is, it's already obvious what they represent. Scratch my above comment 👍